articles/tokens/guides/configure-refresh-token-rotation.md

```diff
@@ -0,0 +1,71 @@
+---
+description: Learn how to configure Refresh Token rotation.
+topics:
+  - tokens
+  - refresh-tokens
+  - refresh-token-rotation
+contentType: how-to
+useCase:
+  - enable-refresh-token-rotation
+  - configure-refresh-token-rotation
+---
+# Configure Refresh Token Rotation
+
+Configure Refresh Token Rotation for each application using the Management API. When Refresh Token Rotation is enabled, the transition for the end-user is seamless. The application uses the previous non-rotating Refresh Token, which has expired and swaps it for a rotating Refresh Token. 
+
+::: note
+Migration scenarios accommodate automatic token revocation when migrating from a non-rotating Refresh Token to a rotating Refresh Token and vice-versa.
+:::
+
+Request a new Refresh Token/Access Token pair when Refresh Token Rotation is enabled.
+
+- Exchanging a non-rotating Refresh Token when Refresh Token Rotation is enabled deletes all the non-rotating tokens issued for the same `client_id`, resource server, and user and tenant.
+- Exchanging a rotating Refresh Token when Refresh Token Rotation is disabled issues a non-rotating Refresh Token and revokes the Rotating Refresh Token family.
+
+The following steps describe how to configure Refresh Token rotation for a SPA, as an example. 
+
+1. Install the latest version of the `auth0-spa-js` SDK.
+
+    ```text
+    npm install @auth0/auth0-spa-js
+    ```
+
```

Comment:
We'll need another step in this sequence. Simply installing the SDK is not enough to start using refresh tokens, as the developer will need to enable the feature on the SDK for it to start sending the `offline_access` scope.

Here's what it looks like for the developer:

```js
const auth0 = await createAuth0Client({
  domain: '<YOUR AUTH0 DOMAIN>',
  client_id: '<YOUR CLIENT ID>',
  audience: '<YOUR API IDENTIFIER>',
  useRefreshTokens: true
});
```

Feel free to use whatever placeholders make sense. The key part is the `useRefreshTokens` parameter is set to `true`.